server/src/universal/conf/delta-sharing-server.yaml.template

```diff
@@ -18,8 +18,9 @@ shares:
       location: "s3a://<bucket-name>/<the-table-path>"
 # Set the host name that the server will use
 host: "localhost"
-# Set the port that the server will listen on
-port: 80
+# Set the port that the server will listen on, default is 8080 because 
```

Comment:
nit: our default value used in code is actually `80.

```
# Set the port that the server will listen on. Note: using ports below 1024 
# may require a privileged user in some operator systems.
```

Reply:
I changed that just now in `ServerConfig.scala`. Maybe you didn't refresh?